javascript/javascript1/week3/WhenWillWeBeThere.js

```diff
@@ -0,0 +1,14 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function countTravelTime(travelInformation){
+    const kmMin = travelInformation.speed/60;
+    const travelTimeInMin = travelInformation.destinationDistance/kmMin;
+    let minutes = travelTimeInMin%60;
+    const hours = (travelTimeInMin - minutes)/60;
+    minutes = Math.round(minutes);
+    return `${hours} hours and ${minutes} minutes`
```

Comment:
nit: there are several edge cases here.
1. What if speed === 0?
2. What if time to travel is less than hour: answer would be 0 hours and XX minutes. Does it makes sense to remove that `0 hours`
3. What if we have 1 hour to travel. We'll have `1 hours`, plural form instead of singular